config/config.exs

```diff
@@ -50,3 +50,14 @@ config :phoenix, :json_library, Jason
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{config_env()}.exs"
+
```

Comment:
This needs to be before the import above in case we want to override anything in another environment